### ISSUE TITLE
Apps are removed/greyed out in an Apex when updating in the App Store

### ISSUE THREAD

**Issue by Zaimojin** [caughtinflux/Apex#10]
iPhone 6 Plus on 9.3.3.

Regardless of location of the Apex group, apps either are fully removed or greyed out after updated through the AppStore.  Only way to fix is to add app back to the apex; if the App is greyed out, it has to be removed from the apex and re-added.

**Comment by caughtinflux** [comment]
Yep I've known about this for a while, will see what I can do. Fwiw, if you tap+hold on an icon that has been thrown out of the Apex stack, it should go right back.
